src/Pokemon.java

```diff
@@ -0,0 +1,68 @@
+import java.util.List;
+
+public class Pokemon {
+    String name;
+    int level;
+    int hp;
+    String food;
+    String sound;
+    String type;
```

Comment:
Deze waarde wordt nooit gezet. De `getType()` methode retruned daarom altijd `null`, waardoor jou applicatie crasht. Je zou deze nog in de constructor moeten zetten